nix/rollup-bridge-contracts.nix

```diff
@@ -1,6 +1,5 @@
 { lib
 , stdenv
-, biome
```

Comment:
why remove `biome`

Reply:
because we are not using it :)

Reply:
Can we skip the linting topic for this PR?
The linting job is failing for me now and it feels not really related to the PR

Reply:
Can we skip the linting topic for this PR?
The linting job is failing for me now and it feels not really related to the PR